traffic_ops/install/bin/postinstall

```diff
@@ -568,8 +558,10 @@ sub getDefaults {
 sub setupDatabase {
     my $todbconf     = shift;
     my $opensslconf  = shift;
-    my $genCert      = shift;
     my $setupMaxmind = shift;
+    my $databaseConfFile = shift;
+
+    my $dbconf = InstallUtils::readJson( $databaseConfFile );
 
```

Comment:
This is a catch-22 -- the db connection must be tested first -- just to make sure the dbconf.yml settings are correct.   Only then can the `db/admin.pl ... setup` do anything....